### PR TITLE
fix: replace `SelectIcon` in `Select` with @yamada-ui/lucide

### LINKED ISSUE #3533: Replace `SelectIcon` with `LucideIcon`
### Description

Replace `SelectIcon` with `LucideIcon`.

### Problem Statement/Justification

Replace the icons from `@yamada-ui/icon` with those from `@yamada-ui/lucide`.

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by mei28**
Closes #3533

## Description

* replace `SelectIcon` in `Select` with `@yamada-ui/lucide`

### CODE REVIEW

File: pnpm-lock.yaml
[the remark this answers is not in the document]
Comment (mei28) [reply pnpm-lock.yaml:1]: The same issue occurred in #3542.

**Review by illionillion — DISMISSED** [review]
LGTM

**Comment by hirotomoyamada** [comment]
@mei28 @illionillion 

Hi, Styles must have the same look and feel.

**Comment by mei28** [comment]
> @mei28 @illionillion
> 
> Hi, Styles must have the same look and feel.

@hirotomoyamada 
Sorry, I chose a `LucideIcon` that I thought had a similar feel to the original icon. Could you let me know specifically which ones are incorrect?

File: packages/components/select/src/select-icon.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const SelectIcon = forwardRef<SelectIconProps, "div">(
         __css={css}
         {...rest}
       >
-        {isValidElement(children) ? cloneChildren : <ChevronIcon />}
+        {isValidElement(children) ? cloneChildren : <ChevronDownIcon />}
```
Comment (mei28) [inline packages/components/select/src/select-icon.tsx:53]:
```suggestion
        {isValidElement(children) ? cloneChildren : <ChevronDownIcon fontSize={'md'} />}
```

Comment (mei28) [reply packages/components/select/src/select-icon.tsx:53]: @hirotomoyamada 
Thank you for your comment. I understand that you are saying the icon I replaced is bigger than the original one. If so, I am planning to make the changes as shown above. Does that work for you?

**Pull request closed without merging**